packages/modules/viewer-react/src/hooks/useFrontstages.tsx

```diff
@@ -54,13 +57,25 @@ export const useFrontstages = (
 
     if (requiresConnection) {
       // we require a connection, so initialize the DefaultFrontstage that contains the views that we want
-      const defaultFrontstageProvider = new DefaultFrontstage(
-        defaultUiConfig,
+      // const defaultFrontstageProvider = new DefaultFrontstage(
```

Comment:
can just delete this unless there is reason to keep it around

Reply:
Thought id leave this around for testing the new ui providers in the todo; was thinking we can remove this chunk once we're at a feature/look parity with whats on master